src/cpu/o3/comm.cc

```diff
@@ -0,0 +1,74 @@
+#include "cpu/o3/comm.hh"
```

Comment:
You should add a license header for the new file

Reply:
I fixed these issues in a new pull request, #1681.